src/plugins/intel_gna/src/gna_plugin_query_api.cpp

```diff
@@ -60,6 +61,21 @@ Parameter GNAPlugin::GetMetric(const std::string& name, const std::map<std::stri
         return GNADeviceHelper::GetGnaLibraryVersion();
     } else if (ov::execution_devices == name) {
         return decltype(ov::execution_devices)::value_type {GetName()};
+    } else if (name == ov::caching_properties) {
+        std::vector<ov::PropertyName> cachingProperties;
+        cachingProperties.push_back(ov::PropertyName(ov::intel_gna::scale_factors_per_input.name(), ov::PropertyMutability::RO));
```

Comment:
Can we have a list of cached properties in one place? Actually the same list is available here:
```
const Parameter Config::GetSupportedProperties(bool compiled) {
    ov::PropertyMutability model_mutability = compiled ? ov::PropertyMutability::RO : ov::PropertyMutability::RW;
```

Reply:
I will change this according your suggestion